docs/reference/deprecated.md

```diff
@@ -4,6 +4,7 @@ Gutenberg's deprecation policy is intended to support backwards-compatibility fo
 
  - All components in `wp.blocks.*` are removed. Please use `wp.editor.*` instead.
  - `wp.blocks.withEditorSettings` is removed. Please use the data module to access the editor settings `wp.data.select( "core/editor" ).getEditorSettings()`.
+ - `isPrivate: true` has been removed. Please use `supports.inserter: false` instead.
```

Comment:
Should we add here some context? E.g.:
``isPrivate: true` has been removed from blocks API.`